.gitlab-ci.yml

```diff
@@ -133,10 +133,10 @@ variables:
   S3_DSD6_URI: s3://dsd6-staging
   RELEASE_VERSION_6: nightly
   RELEASE_VERSION_7: nightly-a7
-  DATADOG_AGENT_BUILDIMAGES: v7341670-fda66f1
+  DATADOG_AGENT_BUILDIMAGES: v7556641-c061240
   DATADOG_AGENT_BUILDERS: v6814105-5ec7b27
   DATADOG_AGENT_WINBUILDIMAGES: v7341670-fda66f1
-  DATADOG_AGENT_ARMBUILDIMAGES: v7341670-fda66f1
+  DATADOG_AGENT_ARMBUILDIMAGES: v7556641-c061240
   DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v7341670-fda66f1
   DATADOG_AGENT_NIKOS_BUILDIMAGES: v7341670-fda66f1
```

Comment:
`DATADOG_AGENT_BUILDIMAGES`, `DATADOG_AGENT_WINBUILDIMAGES`, `DATADOG_AGENT_ARMBUILDIMAGES`, `DATADOG_AGENT_SYSPROBE_BUILDIMAGES` and `DATADOG_AGENT_NIKOS_BUILDIMAGES` used to have the same tag value.
I wonder if we should update them all.

Reply:
Good catch, yes, if possible they should have the same value.